### PR TITLE
Add arm for deployment by tidb-ansible

### PR DESCRIPTION
**Pull request by liubo0127**
### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/ansible.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Ansible 是一款自动化运维工具，[TiDB Ansible](https://github.com/pingc
 1. 部署目标机器若干
 
     - 建议 4 台及以上，TiKV 至少 3 实例，且与 TiDB、PD 模块不位于同一主机，详见[部署建议](/how-to/deploy/hardware-recommendations.md)。
-    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 架构 (amd64)。
+    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 架构 (amd64 或者  arm64)。
```
Comment (sykp241095) [inline how-to/deploy/orchestrated/ansible.md]: @n0vad3v 如果是 arm，需不需要 7.6.1810 ？

Comment (sykp241095) [reply how-to/deploy/orchestrated/ansible.md]: @liubo0127 需不需要强调下 >= 4.0.0-beta.2？

Comment (liubo0127) [inline how-to/deploy/orchestrated/ansible.md]:
```suggestion
    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 (AMD64) 或者 ARM64 架构。
```

File: how-to/deploy/orchestrated/ansible.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Ansible 是一款自动化运维工具，[TiDB Ansible](https://github.com/pingc
 1. 部署目标机器若干
 
     - 建议 4 台及以上，TiKV 至少 3 实例，且与 TiDB、PD 模块不位于同一主机，详见[部署建议](/how-to/deploy/hardware-recommendations.md)。
-    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 架构 (amd64)。
+    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 (AMD64) 或者 ARM64 架构。
```
Comment (liubo0127) [inline how-to/deploy/orchestrated/ansible.md]:
```suggestion
    - 我们支持 x86_64 (AMD64) 和 ARM64 两种架构。在 AMD64 架构下，建议使用 CentOS 7.3 及以上版本 Linux 操作系统；在 ARM 架构下，建议使用 CentOS 7.6 1810 版本 Linux 操作系统。
```

File: how-to/deploy/orchestrated/ansible.md [hunk 3]
```diff
@@ -419,6 +419,8 @@ analyzing CPU 0:
 > **注意：**
 >
 > 请使用内网 IP 来部署集群，如果部署目标机器 SSH 端口非默认的 22 端口，需添加 `ansible_port` 变量，如 `TiDB1 ansible_host=172.16.10.1 ansible_port=5555`。
+>
+> 如果是 arm 架构的机器，需要将 `cpu_architecture` 改为 `arm64`。
```
Comment (TomShawn) [inline how-to/deploy/orchestrated/ansible.md]: ARM

File: how-to/deploy/orchestrated/ansible.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ Ansible 是一款自动化运维工具，[TiDB Ansible](https://github.com/pingc
 1. 部署目标机器若干
 
     - 建议 4 台及以上，TiKV 至少 3 实例，且与 TiDB、PD 模块不位于同一主机，详见[部署建议](/how-to/deploy/hardware-recommendations.md)。
-    - 推荐安装 CentOS 7.3 及以上版本 Linux 操作系统，x86_64 架构 (amd64)。
+    - 我们支持 x86_64 (AMD64) 和 ARM64 两种架构。在 AMD64 架构下，建议使用 CentOS 7.3 及以上版本 Linux 操作系统；在 ARM 架构下，建议使用 CentOS 7.6 1810 版本 Linux 操作系统。
```
Comment (TomShawn) [inline how-to/deploy/orchestrated/ansible.md]: 技术文档中应避免使用第一人称`我们`。

Comment (sykp241095) [reply how-to/deploy/orchestrated/ansible.md]: +1

File: how-to/deploy/orchestrated/ansible.md (hunk 3 above)
Comment (sykp241095) [reply how-to/deploy/orchestrated/ansible.md]: +1

**Pull request merged**